wave_geometry/include/wave/geometry/numerical_test_functors.hpp

```diff
@@ -189,7 +190,7 @@ void numerical_jacobian(FunctorType F,
         auto F_x = F(evaluation_point);
 
         // Finally compute the finite difference.
-        Vec3 finite_difference = (F_xp1 - F_x) / (step_size);
+        VecX finite_difference = (F_xp1 - F_x) / (step_size);
```

Comment:
This is a fixed size right?

Reply:
It is now. I added the se3 transform stuff into the pr initially before remembering that gtsam has its own transform class and took it out again. I left this in because at some point we'll need this to be flexible anyway